cmake/scripts/linux/platform/X11.cmake

```diff
@@ -0,0 +1,2 @@
+set(PLATFORM_REQUIRED_DEPS OpenGl EGL X XRandR LibDRM)
```

Comment:
hmm we have never explicitly set glx as a dependency anywhere I can find. Maybe we need to add a FindGLX.cmake module?